ocaml/xe-cli/bash-completion

```diff
@@ -108,6 +108,9 @@ _xe()
                     host-cpu-*|host-crashdump-*|gpu-group-*|vgpu-type-*|pvs-server-*|pvs-proxy-*|pvs-site-*)
                         # Chop off at the second '-' and append 'list'
                         cmd="$(echo ${OLDSTYLE_WORDS[1]} | cut -d- -f1-2)-list";;
+                    sdn-controller-*)
```

Comment:
Sorry, that's not what I meant. I meant this:

    host-cpu-*|host-crashdump-*|gpu-group-*|vgpu-type-*|pvs-server-*|pvs-proxy-*|pvs-site-*|sdn-controller*)